src/main/java/org/fundacionjala/pivotal/core/api/APIResponseName.java

```diff
@@ -0,0 +1,35 @@
+package org.fundacionjala.pivotal.core.api;
+
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class APIResponseName {
+    private String date;
+
+    /**
+     * Method to get the creation date.
+     * @return the creationdate.
+     */
+    public String getDate() {
+        return date;
+    }
+
+    /**
+     * Method to generate the creation date.
+     * @return the creationdate.
+     */
+    public String generateDate() {
+        Date date = new Date();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "date" which hides the field declared at line 8. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AHiddenFieldCheck)